src/components/confirm/confirm.component.js

```diff
@@ -67,24 +89,29 @@ Confirm.defaultProps = {
   showCloseIcon: false,
   destructive: false,
   iconType: null,
+  cancelButtonType: "secondary",
 };
 
 Confirm.propTypes = {
   ...Dialog.propTypes,
   // ** A custom event handler when a confirmation takes place */
   onConfirm: PropTypes.func.isRequired,
-
   // ** Customise the confirm button label */
   confirmLabel: PropTypes.string,
-
   // ** Customise the cancel button label */
   cancelLabel: PropTypes.string,
-
-  /** Apply destructive style to the button */
+  /** Apply destructive style to the buttons */
   destructive: PropTypes.bool,
-
   /** Defines an Icon type within the button (see Icon for options) */
   iconType: PropTypes.oneOf(["error", "warning"]),
+  /** Makes cancel button disable */
+  disabledCancel: PropTypes.bool,
+  /** Makes confirm button disable */
+  disabledConfirm: PropTypes.bool,
+  /** Allows to setup buttonType into cancel button */
+  cancelButtonType: PropTypes.oneOf(OptionsHelper.buttonTypes),
+  /** Adds isLoading state into confirm button */
+  isLoading: PropTypes.bool,
```

Comment:
I think this should be `loadingConfirm` so it's clear it only applies to the confirm button

Reply:
`isLoadingConfirm` will be fine as well?

Reply:
Yeah that's fine 👍🏻